Assets/HoloToolkit/Input/Scripts/Utilities/BaseControllerPointer.cs

```diff
@@ -10,20 +10,10 @@ public abstract class BaseControllerPointer : AttachToController
     [Range(0f, 360f)]
     protected float CurrentPointerOrientation;
 
-    [SerializeField]
-    [Range(0.5f, 50f)]
-    private float extentOverride = 2f;
```

Comment:
This most def should stay here.

Reply:
This class was added by me in the PR that added the visualization. The visualizer doesn't need to know about the pointer's extent all the time.

Why should it stay?

Reply:
This was present in the MultiPointer Refactor branch, but that's because pointers and pointer visualization were one and the same. Here, they're separate, and tracking the extent in multiple places isn't necessary.

Reply:
A bit busy right now, but I'll be sure to explain in detail later.

Reply:
I would ask if there is actually a good reason to remove the property.  It seems a useful override if you wanted to limit the pointer extent.  Unless this is being removed because it's causing a problem.

Likely this will be re-implemented differently in vNext, so I would vote to keep it in for backwards compatibility for now.

Reply:
The removal is not the `extentOverride` present in [`InputSourcePointer`](https://github.com/keveleigh/HoloToolkit-Unity/blob/PointingRaysExtra/Assets/HoloToolkit/Input/Scripts/Focus/InputSourcePointer.cs#L41). The removed `extentOverride` was added in [my PR adding pointer visualizations](https://github.com/Microsoft/MixedRealityToolkit-Unity/blame/may18_dev/Assets/HoloToolkit/Input/Scripts/Utilities/BaseControllerPointer.cs#L15), to the classes that are solely responsible for the visualizations (and not actual pointing/interaction). It was not actually doing anything, except being [immediately overwritten with the default cursor distance](https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/2201/files/56b9a6ecc6ce59c2ef9d8f5644af3c6f96871cf0#diff-269f5fd5cb2a1585a50c71319e1706b6L312) (again, because this class only cares about visualization, not pointing). I shouldn't have added it in the first place.